### PR TITLE
Duck Pattern Cleaning

### PR DESCRIPTION
**Pull request by SelcukEdma**
Things to be Changed 

- [x] Applied DuckStructure

- [x] Remove files from actions/reducer/sagas fro Search Result

- [x] Added searchResult files into State folder

NOTE:
This is only for `SearchResult`. I will be adding another for RequestAssets

### CODE REVIEW

File: src/Components/Scenes/SearchResults/SearchResults-Container.js [hunk 1]
```diff
@@ -5,8 +5,9 @@ import {
   searchResultRequest,
   setToggleSearchCheckbox,
   setToggleSearchAllCheckbox
-} from '../../../Actions/searchResultActions';
+} from '../../../State/SearchResult/actions';
 import { requestAssetsClick } from '../../../Actions/requestAssetActions';
+// import { requestAssetsClick } from '../../../State/SearchResult/actions'
```
Comment (ed-roh) [inline src/Components/Scenes/SearchResults/SearchResults-Container.js]: unneeded comment

File: src/Reducers/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { combineReducers } from 'redux';
-import searchResultReducer from './searchResultReducer';
+import searchResultReducer from '../State/SearchResult/reducer';
```
Comment (ed-roh) [inline src/Reducers/index.js]: i think for the file names we had was reducers plural just like in the other reducers file names

File: src/State/SearchResult/actions.ts [hunk 3]
```diff
@@ -1,41 +1,26 @@
 import * as types from './types';
 
-export const searchResultPageLoad = params => ({
+export const searchResultPageLoad = (params: any) => ({
   type: types.SEARCH_RESULT_PAGE_LOAD,
   payload: params
 });
 
-export const searchHandleInput = e => ({
+export const searchHandleInput = (e: { target: { value: any } }) => ({
```
Comment (ed-roh) [inline src/State/SearchResult/actions.ts]: I believe there's a React.FormEvent or some type like that we can use

File: src/State/SearchResult/reducer.ts [hunk 4]
```diff
@@ -0,0 +1,143 @@
+import initialState from '../../Reducers/initialState';
```
Comment (ed-roh) [inline src/State/SearchResult/reducer.ts]: file name should be reducers, not reducer just so we can be consistent with the other state files

**Review by ed-roh — DISMISSED** [review]

File: src/Components/Scenes/SearchResults/SearchResults-Container.js (hunk 1 above)
Comment (SelcukEdma) [reply src/Components/Scenes/SearchResults/SearchResults-Container.js]: Removed

File: src/Reducers/index.js (hunk 2 above)
Comment (SelcukEdma) [reply src/Reducers/index.js]: added plural

File: src/State/SearchResult/actions.ts (hunk 3 above)
Comment (SelcukEdma) [reply src/State/SearchResult/actions.ts]: Yeah thats good one. Added React.event

File: src/State/SearchResult/reducer.ts (hunk 4 above)
Comment (SelcukEdma) [reply src/State/SearchResult/reducer.ts]: Idk why its showing single. on my code its plural

**Review by ed-roh — APPROVED** [review]

**Pull request merged**